tahrir/views/explore.py

```diff
@@ -95,6 +97,47 @@ def explore_badges():
     )
 
 
+@bp.route("/json/discover/accolade")
+def json_discover_accolade():
+    all_badges = g.tahrirdb.get_all_badges().all()
+    newest_badges = sorted(all_badges, key=lambda badge: badge.created_on, reverse=True)[:40]
```

Comment:
This is confusing, what we call "newest badges" here.
Given the logic here, the newest badges are just the first 40 badges, given that `tahrirdb.get_all_badges` is ordered by datetime.

No query based on creation_date, awarded, or the like, making it more accurate w/ pagination mechanism.

care to clarify?